spec/umbra/config_spec.rb

```diff
@@ -4,7 +4,7 @@
   describe '.default' do
     let(:config) { described_class.default }
 
-    it { expect(config.publisher).to eq(Umbra::Publisher) }
+    it { expect(config.publisher).to be_a(Umbra::Publisher) }
```

Comment:
`be_an`? not 100% sure how i feel about that

Reply:
🤔 Any suggestions on a better way about this. The only expectation is that the default config is sensible and works.